<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>join_group_pg</class>
 <widget class="QDialog" name="join_group_pg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="create_bt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>加入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_bt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="group_name_j">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入群名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
